pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -56,6 +58,10 @@
   // protection is enabled. If the upload does not finish within the timeout, next upload can override the previous one.
   private static final String SEGMENT_UPLOAD_TIMEOUT_IN_MILLIS = "controller.segment.upload.timeoutInMillis";
 
+  // Defines the kind of storage and the underlying PinotFS implementation
+  private static final String PINOT_FS_FACTORY_CLASS_PREFIX = "controller.storage.factory.class";
+  private static final String PINOT_FS_FACTORY_CLASS_LOCAL = "controller.storage.factory.class.file";
```

Comment:
controller.storage.factory.class.file reads like this is a file name.

How about "controller.storage.factory.class.localFS"?
Also, this is default, and does not need any additional config (or, are we planning to migrate the configs for segmentDir etc underneath this setting)?

Reply:
no, this is only for different kinds of filesystems. For each fs we configure (for example, internally, we can configure dali), we will have one of these strings. Basically, this will jsut like the immutable map in segment fetcher